src/components/footnote/footnote.scss

```diff
@@ -0,0 +1,46 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+
+$footnote-offset: ms.step(-2);
+$footnote-offset-negative: calc(#{$footnote-offset} * -1);
+$footnote-offset-left-negative: calc(
+  #{$footnote-offset-negative} + #{ms.step(1)} * -1
+);
+
+/// Footnotes are list items with return links after their content
+///
+/// 1. Since footnotes are linked to using hash links, we give them a little
+///    vertical offset so the text isn't right at the top of the screen
+.c-footnote {
+  position: relative;
+  scroll-margin: $footnote-offset; /* 1 */
+
+  /// When a footnote is target by the URL hash we apply a background color to it
+  /// We use an absolutely positioned pseudo-element so our background will extend
+  /// beneath the list item ::marker
+  &:target::before {
+    background-color: color.$base-gray-lighter;
+    border-radius: size.$border-radius-medium;
+    bottom: $footnote-offset-negative;
+    content: '';
+    left: $footnote-offset-left-negative;
+    position: absolute;
+    right: $footnote-offset-negative;
+    top: $footnote-offset-negative;
+    z-index: -1;
+  }
+
+  /// We apply some basic styles to the return link.
+  /// These are so the links get nice focus styles.
+  .c-footnote-link-item__return-link {
```

Comment:
The centering feels slightly off. Do you agree? 🤔 

<img width="346" alt="Screen Shot 2021-09-02 at 10 11 12 AM" src="https://user-images.githubusercontent.com/459757/131890731-5cfa606d-1c81-4bf2-8c05-64c94415a345.png">

Reply:
I do... lemme see what I can do

Reply:
How does this feel @gerardo-rodriguez ?
<img width="894" alt="Screen Shot 2021-09-02 at 3 10 19 PM" src="https://user-images.githubusercontent.com/5798536/131922838-8786ef7c-fe0c-4d96-ad75-e12004766434.png">

(Ended up being kinda tricky... pushed in https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/ec4095f71866ffe55b342526e2a76557a2da81c4)

Reply:
@Paul-Hebert Much better, thank you! 🎉